src/ui/SimpleFilter/SimpleFilter.ts

```diff
@@ -458,6 +470,21 @@ export class SimpleFilter extends Component {
     return this.circleElement.el;
   }
 
+  public buildClearElement(): HTMLElement {
+    this.clearElement = $$('button', { className: 'coveo-simplefilter-eraser' }, SVGIcons.icons.mainClear);
+    this.clearElement.hide();
+
+    this.clearElement.setAttribute('aria-label', l('Clear', this.options.title));
+    this.clearElement.setAttribute('title', l('Clear', this.options.title));
+
+    this.clearElement.on('click', (evt: Event) => {
```

Comment:
There is a class called `AccessibleButton` that accepts the root element and makes it easy to add click handlers, keypress "enter" handlers so that the button is keyboard accessible too.

Reply:
I was using it but @olamothe pointed out that a standard button with appropriate aria-labels will suffice, as `AccessibleButton` was a workaround when it was not possible to change the markup of existing elements and still accessibility was required.

Reply:
Okay, sounds good, the most important is to ensure that the clear button is keyboard accessible in addition to a mouse click.

Reply:
It is ;-)